openssl-sys/src/handwritten/evp.rs

```diff
@@ -260,6 +260,7 @@ const_ptr_api! {
 extern "C" {
     pub fn EVP_CIPHER_CTX_new() -> *mut EVP_CIPHER_CTX;
     pub fn EVP_CIPHER_CTX_free(ctx: *mut EVP_CIPHER_CTX);
+    pub fn EVP_MD_CTX_copy(dst: *mut EVP_MD_CTX, src: *const EVP_MD_CTX) -> c_int;
```

Comment:
According to the docs, it seems like EVP_MD_CTX_copy_ex should be used instead of EVP_MD_CTX_copy.

Reply:
Fair enough, missed that comment, will update